src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimantClaimMatcher.java

```diff
@@ -440,16 +440,27 @@ public boolean trialArrangementsSubmitted() {
         Optional<LocalDateTime> eventTime = getTimeOfMostRecentEventOfType(
             EnumSet.of(CaseEvent.GENERATE_TRIAL_READY_FORM_APPLICANT));
         Optional<LocalDateTime> orderTime = getTimeOfLastNonSDOOrder();
+
         return caseData.isFastTrackClaim()
             && caseData.getTrialReadyApplicant() != null
-            && !ListingOrRelisting.RELISTING.equals(caseData.getListingOrRelisting())
+            && ((nonNull(caseData.getListingOrRelisting()) && !ListingOrRelisting.RELISTING.equals(
+                caseData.getListingOrRelisting())) || isAutomaticHearingNotModifiedAfterTrialNotified())
             && (CaseState.HEARING_READINESS.equals(caseData.getCcdState()) || CaseState.PREPARE_FOR_HEARING_CONDUCT_HEARING.equals(caseData.getCcdState()))
             && !isBundleCreatedStatusActive()
             && isHearingLessThanDaysAway(DAY_LIMIT)
             && (eventTime.isPresent())
             && (orderTime.isEmpty() || eventTime.get().isAfter(orderTime.get()));
     }
 
+    private boolean isAutomaticHearingNotModifiedAfterTrialNotified() {
+        Optional<LocalDateTime> automaticHearingRequested = Optional.ofNullable(caseData.getHearingRequestedAHN());
```

Comment:
I don't think you need to create a field hearingRequestedAHN.
I've updated getWhenWasHearingScheduled in this PR, so it checks for either HearingScheduled or HMC hearing notice events:
https://github.com/hmcts/civil-service/pull/5987/files

You can create a new method that just checks for time of HMC Hearing notice event, then you don't need to save the date in case data.

Reply:
updated